pages/en/contrib/Reporting-issues.md

```diff
@@ -41,6 +41,10 @@ There are three types of issues:
 
 ### Security issues
 
+We take security vulnerabilities in LoopBack very seriously for our users.
+
+It is recommended to never debug log anything in production that may contain sensitive data. For example, do not log the data payload for `PersistedModel.create` requests as it may contain sensitive data sent from the client.
```

Comment:
This is not the advice I had in mind.

We are recommending to never turn on debug logs on production, because debug logs often contain sensitive data sent by the client.

It is ok to debug-log the payload of `PersistedModel.create` if debug logs are used only at development time.